live-examples/js-examples/promise/promise-finally.js

```diff
@@ -0,0 +1,17 @@
+function hasMailArrived() {
+  return new Promise((resolve, reject) => {
+    (Math.random() > 0.5 && resolve('Mail has arrived')) ||
+        reject(new Error('Failed to arrive'));
```

Comment:
I'm not sure it's my place to "correct" but that syntax feels a bit too fancy. What about:
```suggestion
        if (Math.random() > 0.5) {
          resolve('Mail has arrived');
        } else {
          reject(new Error('Failed to arrive'));
        }
```

Reply:
Clearly, I don't know how to make suggestions here in the UI. But I hope it makes sense.

Reply:
> Clearly, I don't know how to make suggestions here in the UI. But I hope it makes sense.

Yeah it made sense no worries! 😸